### DISCUSSION TITLE
Unable to reset password

### DISCUSSION THREAD

**Discussion by jc-gomes** [community/community#34268] [Other Feature Feedback, Questions, & Ideas]
I have a Github account that I havnt accessed in a while and have forgotten the password to. I am unable to reset the password as the email address that the account was tied to was never verified. I tried contacting Github support and its been almost two weeks with no response. What are my options here?

**Comment by GempitaRizki** [comment]
try again to contact support from github , because there is no specific answer how to get your account instantly. CMIIW

**Reply by jc-gomes** [reply]
Emailed them 3 times. No responses yet..

**Comment by Wep-wep** [comment]
Intente con un número de telefono

**Reply by jc-gomes** [reply]
I did not see a support phone number for them.

**Comment by radar-blip** [comment]
You said you emailed them, have you tried this: [Support page - Accounts](https://support.github.com/contact/account)

**Reply by jc-gomes** [reply]
Yeah that is the support page I used.

**Reply by radar-blip** [reply]
Okay, just checking. 
Sometimes it takes a while for them to get back to you. 
They have lots of questions to answer, lots of problems to solve.

**Reply by jc-gomes** [reply]
Understandable. It's just a little concerning as its been a while with no response and I really need access to my other account.

**Reply by radar-blip** [reply]
I totally understand. Sorry I cant help you any further. Hang in there.

**Reply by jc-gomes** [reply]
Appreciate the response. Thank you.
